feedservice/redis/redis.go

```diff
@@ -0,0 +1,68 @@
+package redis
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gomodule/redigo/redis"
+)
+
+const (
+	NVT_FILENAME_POS           = 0
+	NVT_REQUIRED_KEYS_POS      = 1
+	NVT_MANDATORY_KEYS_POS     = 2
+	NVT_EXCLUDED_KEYS_POS      = 3
+	NVT_REQUIRED_UDP_PORTS_POS = 4
+	NVT_REQUIRED_PORTS_POS     = 5
+	NVT_DEPENDENCIES_POS       = 6
+	NVT_TAGS_POS               = 7
+	NVT_CVES_POS               = 8
+	NVT_BIDS_POS               = 9
+	NVT_XREFS_POS              = 10
+	NVT_CATEGORY_POS           = 11
+	NVT_FAMILY_POS             = 13
```

Comment:
fix ordinal
```suggestion
	NVT_FAMILY_POS             = 12
	NVT_NAME_POS               =  13
```

Reply:
But the timeout Value is still written in redis and has the index 12. If I adjust the numbers they don't match their corresponding redis entry

Reply:
No, it is not stored anymore in the key. Under the `nvt:<OID>`, there are values from 0-13, and there is no timeout anymore. Timeout is now stored under `oid:<OID>:prefs` , with pref ID 0.

Check
-  https://github.com/greenbone/gvm-libs/blob/05845e35f5e87fe8f513c5d64d3df35c3540b3fe/util/kb.h#L72
- https://github.com/greenbone/gvm-libs/blob/05845e35f5e87fe8f513c5d64d3df35c3540b3fe/util/kb.c#L1561

```
$ redis-cli -s /tmp/redis-openvas.sock 
redis /tmp/redis-openvas.sock> SELECT 1
OK
redis /tmp/redis-openvas.sock[1]> lrange nvt:1.3.6.1.4.1.25623.1.0.902269 0 -1
 1) "2010/secpod_ms10-012-remote.nasl"
 2) ""
 3) "Host/runs_windows"
 4) "SMB/samba"
 5) ""
 6) "445"
 7) "netbios_name_get.nasl, smb_nativelanman.nasl, os_detection.nasl"
 8) "last_modification=1578387992|creation_date=1287755515|cvss_base_vector=AV:N/AC:L/Au:N/C:C/I:C/A:C|impact=Successful exploitation will allow remote attackers to execute arbitrary\n  code or cause a denial of service or bypass the authentication mechanism via brute force technique.|affected=- Microsoft Windows 7\n\n  - Microsoft Windows 2000 Service Pack and prior\n\n  - Microsoft Windows XP Service Pack 3 and prior\n\n  - Microsoft Windows Vista Service Pack 2 and prior\n\n  - Microsoft Windows Server 2003 Service Pack 2 and prior\n\n  - Microsoft Windows Server 2008 Service Pack 2 and prior|insight=- An input validation error exists while processing SMB requests and can\n  be exploited to cause a buffer overflow via a specially crafted SMB packet.\n\n  - An error exists in the SMB implementation while parsing SMB packets during\n  the Negotiate phase causing memory corruption via a specially crafted SMB packet.\n\n  - NULL pointer dereference error exists in SMB while verifying the 'share'\n  and 'servername' fields in SMB packets causing denial of service.\n\n  - A lack of cryptographic entropy when the SMB server generates challenges\n  during SMB NTLM authentication and can be exploited to bypass the authentication mechanism.|solution=The vendor has released updates. Please see the references for more information.|summary=This host is missing a critical security update according to\n  Microsoft Bulletin MS10-012.|qod_type=remote_app|solution_type=VendorFix"
 9) "CVE-2010-0020, CVE-2010-0021, CVE-2010-0022, CVE-2010-0231"
10) ""
11) "URL:http://support.microsoft.com/kb/971468, URL:http://www.vupen.com/english/advisories/2010/0345, URL:https://docs.microsoft.com/en-us/security-updates/securitybulletins/2010/ms10-012"
12) "4"
13) "Windows : Microsoft Bulletins"
14) "Microsoft Windows SMB Server NTLM Multiple Vulnerabilities (971468)"

```